server/api/config.go

```diff
@@ -36,6 +38,29 @@ func (h *confHandler) Get(w http.ResponseWriter, r *http.Request) {
 	h.rd.JSON(w, http.StatusOK, h.svr.GetConfig())
 }
 
+func (h *confHandler) Post(w http.ResponseWriter, r *http.Request) {
+	config := h.svr.GetConfig()
+	data, err := ioutil.ReadAll(r.Body)
+	r.Body.Close()
+	if err != nil {
+		h.rd.JSON(w, http.StatusInternalServerError, err.Error())
+		return
+	}
+	err = json.Unmarshal(data, &config.Schedule)
```

Comment:
can we support `json.Unmarshal(data, &config)` later? 

IMO, I prefer check key-value one by one.